.pipelines/templates/aks-setup.yaml

```diff
@@ -6,21 +6,44 @@ steps:
     condition: and(succeeded(), eq(variables['CLUSTER_TYPE'], 'aks'))
 
   - script: |
-      az group create -n ${AZURE_CLUSTER_NAME} -l ${AZURE_LOCATION}
+      az group create -n ${AZURE_CLUSTER_NAME} -l $(AZURE_LOCATION)
 
-      az aks create \
-        -g ${AZURE_CLUSTER_NAME} \
-        -n ${AZURE_CLUSTER_NAME} \
-        --subscription ${SUBSCRIPTION_ID} \
-        --node-count ${AGENT_COUNT} \
-        --node-vm-size Standard_D2_v3 \
-        --location ${AZURE_LOCATION} \
-        --service-principal ${CLIENT_ID} \
-        --client-secret ${CLIENT_SECRET} \
-        --windows-admin-username azureuser \
-        --windows-admin-password ${WINDOWS_ADMIN_PASSWORD} \
-        --network-plugin azure \
-        --no-ssh-key
+      if [[ ${LARGE_CLUSTER:-} == "true" ]]; then
+        az aks create \
+          -g ${AZURE_CLUSTER_NAME} \
+          -n ${AZURE_CLUSTER_NAME} \
+          --node-count $(AGENT_COUNT) \
+          --nodepool-name nodepool1 \
+          --node-vm-size Standard_DS2_v2 \
+          --location $(AZURE_LOCATION) \
+          --service-principal $(AAD_CLIENT_ID) \
+          --client-secret $(AAD_CLIENT_SECRET) \
+          --no-ssh-key \
+          --load-balancer-sku standard \
+          --network-plugin azure \
+          --max-pods $(MAX_PODS) \
+          --load-balancer-managed-outbound-ip-count 6
+          MASTERIP=$(az aks show \
+            -g ${AZURE_CLUSTER_NAME} \
+            -n ${AZURE_CLUSTER_NAME} \
+            --query 'fqdn' -o tsv)
+          echo "##vso[task.setvariable variable=MASTERIP]${MASTERIP}"
+          echo "##vso[task.setvariable variable=MASTERINTERNALIP]${MASTERIP}"
+      else
+        az aks create \
+          -g ${AZURE_CLUSTER_NAME} \
+          -n ${AZURE_CLUSTER_NAME} \
+          --subscription $(SUBSCRIPTION_ID) \
+          --node-count $(AGENT_COUNT) \
+          --node-vm-size Standard_D2_v3 \
```

Comment:
node skus are different (Standard_DS2_v2 vs Standard_D2_v3)

Reply:
The load tests and CI tests are run in different subs. The load test sub doesn't have `Standard_D2_v3` skus. I'll update to use `Standard_DS2_v2`.